extensions/2.0/Khronos/KHR_animation_pointer/README.md

```diff
@@ -0,0 +1,282 @@
+# KHR\_animation_pointer
+
+## Contributors
+
+* Alexey Knyazev
+* Ben Houston, Threekit
+* Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+* Don McCurdy
+* Ed Mackey, AGI @emacke
+* Felix Herbst, prefrontal cortex [@hybridherbst](https://twitter.com/hybridherbst)
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+* Joe Herdman, [@jbherdman](https://github.com/jbherdman)
+* Lewis Weaver, Microsoft
+* Marco Hutter
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+Copyright (C) 2018-2022 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension is based on the animation features of glTF 2.0. The structure of the schemas stay the same.  
+  
+The only major addition is, that the output values are mapped using a [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901).
+  
+### Motivation
```

Comment:
Could you please elaborate on the usecases for this extension and provide some samples that shows the benefits of adding this extension?

Reply:
I believe the expanded examples below address this. When we get a model in the Sample Assets repo it'll be even more clear - maybe something with an animated texture transform or color.